2018 Robot/src/org/usfirst/frc/team4999/robot/commandgroups/bluealliance/PlaceCubeOnScaleLeft.java

```diff
@@ -0,0 +1,23 @@
+package org.usfirst.frc.team4999.robot.commandgroups.bluealliance;
+
+import org.usfirst.frc.team4999.commands.autonomous.ForwardBackward;
+import org.usfirst.frc.team4999.commands.autonomous.TurnByDeg;
+import org.usfirst.frc.team4999.robot.commands.lift.MoveLift;
+import org.usfirst.frc.team4999.robot.commands.lift.SetLiftToZero;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+/**
+ *
+ */
+public class PlaceCubeOnScaleLeft extends CommandGroup {
+
+    public PlaceCubeOnScaleLeft() {
```

Comment:
This should have a different sequence depending on which side of the lift we’re aiming for

Reply:
Just one command, takes side as an input, and uses a switch statement